test/test-class-cake.js

```diff
@@ -4,8 +4,8 @@ import { makeClassCake } from '../lib/layer-cake';
 
 test('cajita-wobbly-point-class test', t => {
   try {
-    function* BasePointLayer(x, y) {
-      const [self] = yield {
+    function BasePointLayer(x, y) {
+      return self => ({
```

Comment:
Harden the returned record. See https://github.com/agoric-labs/layer-cake/issues/5